### ISSUE TITLE
Compiled list of Community Suggestions

### ISSUE THREAD

**Issue by warlordjones** [TheCricket/Chisel-2#316]
* Basalt mod compatbility
* Block including the end portal renderer
* Logs
* Hardened Clay
* Quartz
* Lawn/Grass
* Chiseled Silverwood/Greatwood planks
* Gravel, including the pre 1.4 texture

**Comment by Drullkus** [comment]
The end portal renderer was my suggestion a looooong time ago...

**Comment by tterrag1098** [comment]
@warlordjones The end portal render is abhorrently inefficient, just a few blocks would cause noticeable render lag. It is not meant for decorative blocks, just ask @TheCricket. 

Also, use check boxes :P I'll edit it for you

**Comment by Entoarox** [comment]
@tterag1098 You might want to ask @RWTema, He recreated the portal renderer using shaders to allow for its use with the warp itemducts without causing any noticeable lag. (And I think that tesseracts are now also using that shader for the effect)

**Comment by warlordjones** [comment]
Yes, the shader requires a TESR however, meaning decorative blocks using it would need to have a Tile Entity, effectively turning client lag into server lag. Its fine for use in the itemducts because they use TEs anyway, and shouldn't be used in the numbers Chisel blocks generally are.

**Comment by Entoarox** [comment]
@warlordjones Excuse me if I'm completely wrong in my thoughts here, but cant you make it so the tile-entity only exists on the client-side, since the goal here is visuals only, having the client believe a tile entity to exist that isnt actually there should not be a huge problem, or would it be?

**Comment by Drullkus** [comment]
Still not feasible in terms of how TEs work, I think

**Comment by TheCricket** [comment]
Yeah TileEntities cannot be client side only

**Comment by Entoarox** [comment]
That is a shame, and there is absolutely no way for a block to have shaders applied to it without it also being a tile entity?

**Comment by TheCricket** [comment]
Well the thing is, its really laggy I made a block that does this and only rendered the side you were looking at and it still dropped frames a lot

**Comment by tterrag1098** [comment]
Because you were using mojang' s crappy render, not a shader.

**Comment by Drullkus** [comment]
I concur with @tterrag1098 

There's a difference between using the CPU to render vs using GLSL to render

**Comment by TheCricket** [comment]
Beacons check http://prntscr.com/7k2bkx

**Comment by RCXcrafter** [comment]
> - [x] ~~UBC compat.~~

Does that mean that UBC compatibility won't be added?

**Comment by TheCricket** [comment]
We won't add it but they sure can. Its super easy to do it

**Comment by RCXcrafter** [comment]
I started working on a bridge mod, so I just wanted to know for sure.

**Comment by TheCricket** [comment]
I saw
